test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -288,7 +323,14 @@ int run_test_with_two_queue(cl_context &context, cl_command_queue &cmd_queue1,
 
                 if (iter != (maxIter - 1))
```

Comment:
This iter check only needs to be done when !use_fence.  And if you make the change above, we already called finish or wait for events.  I suggest something like if (!use_fence && (iter != (maxIter - 1)))...